Dockerfile

```diff
@@ -1,5 +1,6 @@
 # Build the manager binary
-FROM --platform=${BUILDPLATFORM:-linux/amd64} docker.io/library/golang:1.22.6 AS builder
+FROM --platform=${BUILDPLATFORM:-linux/amd64} brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.22 AS builder
```

Comment:
The base image for upstream should not change. This should be updated during release.

Reply:
Everyone seems to solve this differently. We could maintain separate containerfiles for Konfux and use that in the pipeline params, or we could use `BUILD_ARGS` to pass in a variable that sets the base image.

Reply:
Right, I think we can leverage separate [dockerfiles in the pipeline params](https://github.com/project-koku/koku-metrics-operator/blob/0ec274a9e1c7f3b581018dbf29870348156761a1/.tekton/koku-metrics-operator-pull-request.yaml#L29).  But we can revisit this once we have downstream builds.